js/main.js

```diff
@@ -72,7 +102,7 @@ maps.forEach(map_ => {
                     fillOpacity: 1
                 };
             }
-            baseMaps[key_] = tile_
+            baseMaps[key_] = tile_     
```

Comment:
You should check your code after raising the PR to make sure no unnecessary changes are part of it. You can do so by clicking on `Files Changed` on that PR.